src/discussions/comments/CommentsView.jsx

```diff
@@ -71,11 +71,14 @@ function DiscussionCommentsView({
     ...filterPosts(comments, 'unendorsed')], [comments]);
   return (
     <>
+      {!isLoading
```

Comment:
Did you check a use case where if a user has more than 10 responses like 13 responses and the user clicks on the load more button and an API call fetches new responses?
I think, there would be a UI glitch because when a user will click on load more button,  isLoading will be true and the response will disappear.

Reply:
no, I didn’t. Let me check this use case